905.sort-array-by-parity.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/sort-array-by-parity/
+ */
+
+/**
+ * @param {number[]} nums
+ * @return {number[]}
+ */
+var sortArrayByParity = function(nums) {
+    
+    let evenNumber = nums.filter((a) => a % 2 === 0);
+    let oddNumber = nums.filter((a) => a % 2 === 1);
```

Comment:
```suggestion
    const evenNumbers = nums.filter((a) => a % 2 === 0);
    const oddNumbers = nums.filter((a) => a % 2 === 1);
```
const seems better.
"evenNumber**s**" and "oddNumber**s**" are more self-descriptive.

Reply:
I corrected them